src/plugins/gui/providers/moonpayProvider.ts

```diff
@@ -21,7 +21,7 @@ const pluginDisplayName = 'Moonpay'
 
 const allowedCurrencyCodes: FiatProviderAssetMap = { crypto: {}, fiat: {} }
 const allowedCountryCodes: { [code: string]: boolean } = {}
-const allowedPaymentTypes = { applepay: true, credit: true, googlepay: true }
+const allowedPaymentTypes: { [payment: string]: boolean } = { applepay: true, credit: true, googlepay: false, iach: true }
```

Comment:
Same thing with the type definition hiding info here.

Reply:
Fixed